### DISCUSSION TITLE
Extension error on package

### DISCUSSION THREAD

**Discussion by shero-lim** [microsoft/vscode-discussions#502] [Extension Development QnA] [answered]
when i package the extensions, shows error, how to resolve it?

![image](https://user-images.githubusercontent.com/38414433/222646949-47cfb1af-e00d-4fe5-88fe-3a421efd61ca.png)

**Comment by r3m0t** [comment, marked as answer]
Did you fork one of the vscode-extension-samples, or did you use the yo generator? Can you share your tsconfig and the code in your extension that imports `fetch` and makes http requests?

**Reply by shero-lim** [reply]
I delete @types/isomorphic-fetch and @types/react-router-dom on global node_modules, then reinstall in project, and resolve this issue
